setup.py

```diff
@@ -9,6 +10,21 @@
     changes = f.read()
 
 
+install_requires = [
+    'pytest',
+    'pep8',
+    'pyflakes',
+    'coverage',
+    'sphinx',
+    'pytest-cov',
+]
+
+if version_info.major == 2:
+    install_requires.append([
+        'mock',
+        'future',
+    ])
```

Comment:
n.b. "run more slowly" refers to the dependency collection appearing to take longer. If the deps are there it seems pretty similar, and I didn't measure it.